From f8983d1c99ea4e81e7a59dd48c11287a0238d282 Mon Sep 17 00:00:00 2001
From: Christian Hewitt <christianshewitt@gmail.com>
Date: Wed, 8 Dec 2021 15:28:50 +0000
Subject: [PATCH 54/74] WIP: dt-bindings: arm: amlogic: add support for Beelink
 Mini MXIII

The Beelink Mini MXIII is an Android STB based on the Amlogic P200
(GXBB) reference design with an S905 chip.

Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 Documentation/devicetree/bindings/arm/amlogic.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/arm/amlogic.yaml b/Documentation/devicetree/bindings/arm/amlogic.yaml
index 7e801f208863..096c3cfadeb0 100644
--- a/Documentation/devicetree/bindings/arm/amlogic.yaml
+++ b/Documentation/devicetree/bindings/arm/amlogic.yaml
@@ -56,6 +56,7 @@ properties:
           - enum:
               - amlogic,p200
               - amlogic,p201
+              - beelink,mini-mxiii
               - friendlyarm,nanopi-k2
               - hardkernel,odroid-c2
               - minix,neo-u1
-- 
2.17.1

